.github/workflows/ci.yml

```diff
@@ -41,3 +41,11 @@ jobs:
       with:
         flags: unittests
         fail_ci_if_error: true
+
+    - name: Publish Provider Verification Results
+      if: matrix.python-version == '3.8' && matrix.toxenv == 'django32'
+      run: |
+        export PACT_BROKER_BASE_URL=https://edx.pactflow.io
+        export PUBLISH_VERSION=`git rev-parse --short HEAD`
+        export PUBLISH_VERIFICATION_RESULTS=true
+        pytest edxval/pacts/verify_pact.py --ds=edxval.settings.pact
```

Comment:
This job should be run separately and not part of unit tests. In VEM, publish pact job is run with unit tests because the pact is created/updated during the unit test suite. The same is not true for verification. This job will read pact from the broker/local file and then run the verification.